File: Tareas Java/Impresion Arreglo VB/Impresion Arreglo VB/FormResultado.vb
Public Class FormResultado

    Public filas, columnas As Integer
    Public Arreglo As String(,)
    Public Tabulaciones As Integer()

    Private Sub FormResultado_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Dim fila, tabCadena As String
        Dim TAB, i, j, k As Integer


        For i = 0 To filas - 1

            fila = ""       'Fila que será impresa


            For j = 0 To columnas - 1


                tabCadena = ""

                'Calculamos el tab para cada palabra
                'Sumamos dos para que ninguna palabra esté pegada a la siguiente columna
                TAB = Tabulaciones(j) - Arreglo(i, j).Length + 2


                For k = 0 To TAB - 1

                    tabCadena = tabCadena + " "

                Next


                fila = fila + Arreglo(i, j) + tabCadena

            Next

            lblResultado.Text = lblResultado.Text + fila + vbNewLine

        Next
    End Sub
End Class

File: Tareas Java/Impresion Arreglo/Impresion Arreglo/FormResultado.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Impresion_Arreglo
{
    public partial class FormResultado : Form
    {
        public FormResultado()
        {
            InitializeComponent();
        }

        public int filas, columnas;
        public String[,] Arreglo;
        public int[] Tabulaciones;

        private void FormResultado_Load(object sender, EventArgs e)
        {
            String fila, tabCadena;
            int tab, i, j, k;


            for (i = 0; i < filas; i++){
        
                fila = "";		//Fila que será impresa


                for ( j = 0; j < columnas; j++){


                    tabCadena = "";

                    //Calculamos el tab para cada palabra
                    //Sumamos dos para que ninguna palabra esté pegada a la siguiente columna
                    tab = Tabulaciones[j] - Arreglo[i, j].Length + 2;


                    for ( k = 0; k < tab; k++) {

                        tabCadena = tabCadena + " ";

                    }


                    fila = fila + Arreglo[i, j] + tabCadena;

                }

                lblResultado.Text = lblResultado.Text + fila + "\n";

            } 
        }

    }
}
